canary/canary.properties

```diff
@@ -0,0 +1,14 @@
+# Canary configurations
```

Comment:
This is interesting. Do we want to convert the LH Server to use this? It seems more "java" than what we do with the server. I think it would be best to use one standard for everything...

On the other hand, the dashboard doesn't do this, and this doesn't work well for env vars.